uploader.json

```diff
@@ -507,6 +507,11 @@
             "pattern": [],
             "symbolic_name": "lvs_noheadings"
         },
+        {
+            "command": "/usr/bin/md5sum /dev/null /etc/pki/{product,product-default}/69.pem 2>/dev/null",
```

Comment:
@lonicerae the client will not redirect STDERR in your command, your output will look like this:
```
d41d8cd98f00b204e9800998ecf8427e  /dev/null
/usr/bin/md5sum: /etc/pki/product/69.pem: No such file or directory
/usr/bin/md5sum: /etc/pki/product-default/69.pem: No such file or directory
```